src/cli.ts

```diff
@@ -101,11 +138,21 @@ export function cliWarn(message: string, fromVercelCli = false): void {
  * the function also removes extra indentation on the message allowing us to indent the messages
  * in the code as we please (see https://www.npmjs.com/package/dedent-tabs)
  */
-function prepareCliMessage(message: string, fromVercelCli: boolean): string {
-	return dedent(message)
+function prepareCliMessage(
+	message: string,
+	{ fromVercelCli, styleFromater, spaced }: PrepareCliMessageOptions
+): string {
+	const preparedMessage = dedent(message)
 		.split('\n')
-		.map(line => `${getCliPrefix(fromVercelCli)} ${line}`)
+		.map(
+			line =>
+				`${getCliPrefix(fromVercelCli)} ${
+					styleFromater ? styleFromater(line) : line
+				}`
+		)
 		.join('\n');
+
+	return spaced ? `\n${preparedMessage}\n` : preparedMessage;
```

Comment:
I think first variant is better.
![image](https://user-images.githubusercontent.com/94276879/232110958-fce4a8da-f59e-4335-bdac-736861d80f82.png)
![image](https://user-images.githubusercontent.com/94276879/232112150-75acc28e-df77-48fd-a160-bc3c77bc6b38.png)